kops/kops.go

```diff
@@ -121,25 +120,15 @@ func NewKops() (*KopsCmd, error) {
 //}
 
 func (k *KopsCmd) ReplaceCluster(cluster clusteroperatorv1alpha1.ClusterSpec) error {
```

Comment:
Not a blocking suggestion for this PR, but for future unit-testing it would be better to have an interface between KopsCmd and the caller. Maybe something to add in a future PR if you want to get this one in right away